sdk/communication/Azure.Communication.Chat/tests.yml

```diff
@@ -0,0 +1,17 @@
+trigger: none
```

Comment:
Is there a plan to also remove the tests.yml file in the root of the communications service folder?

Reply:
Yes, that is the plan once we create the new pipelines and make sure that everything is working as expected